src/screens/Departures/PlaceScreen.tsx

```diff
@@ -44,6 +45,14 @@ export default function PlaceScreen({
     date: new Date().toISOString(),
   });
 
+  const {state} = useStopsDetailsData([place.id]);
+
+  if (state.data && place.quays === undefined) {
+    place = state.data.stopPlaces[0];
+  }
```

Comment:
Stemmer det at returverdi fra `useStopsDetailsData` kun er relevant hvis innsendt `Place` ikke inneholder `quays`? Altså at man kan spare seg noen http-kall hvis man snur litt om på rekkefølgen her? Skjønner at det kan bli litt knot for å komme seg rundt regelen om at hooks ikke kan komme etter conditionals, men tror det kan være verdt det.